File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.py
def solution(phone_number):
    # "*"*(len(phone_number)-4)
    # return ''.join(["*" for _ in phone_number[:len(phone_number)-4]]) + phone_number[len(phone_number)-4:]
    return "*"*(len(phone_number)-4) + phone_number[len(phone_number)-4:]

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        //char array로 변경하고 인덱스 접근
        char[] ca = phone_number.toCharArray();
        for(int i=0; i<phone_number.length()-4; i++){
            ca[i] = '*';
        }
        
        //다시 string으로 변경
        String answer = new String(ca);
        return answer;
    }
}